core/modules/widgets/element.js

```diff
@@ -31,9 +31,7 @@ ElementWidget.prototype.render = function(parent,nextSibling) {
 	this.computeAttributes();
 	// Neuter blacklisted elements
 	this.tag = this.parseTreeNode.tag;
-	if($tw.config.htmlUnsafeElements.indexOf(this.tag) !== -1) {
-		this.tag = "safe-" + this.tag;
-	}
+	this.tag = $tw.utils.makeTagNameSafe(this.tag, "safe" + this.tag);
```

Comment:
The original code here was attempting to sanitise the tag name provided by the user. This is in contrast to the behaviour of the helper function which is to fall back to a default if the provided tag name is not valid.

I think one approach or there other should be used consistently across the board. I'd favour the sanitising approach.

Reply:
The logic does only sanitise the tag if it is not valid. That's exactly, what the new code does